db/migrate/20181114213758_create_backup_code_configurations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBackupCodeConfigurations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :backup_code_configurations do |t|
+      t.integer :user_id, null: false
+      t.string :code_fingerprint, default:'', null: false
+      t.string :encrypted_code, default:'', null: false
+      t.boolean :used, default: false
+      t.timestamp :used_at
```

Comment:
Why do we need a `used_at` *and* a `used`? If it's `used_at`, it should be `used`.